### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by gautamborah-okta**
## Description:
- **What's changed?** 

Adding UserType policy rule condition.
https://oktainc.atlassian.net/browse/OKTA-386724

- **Is this PR related to a Monolith release?** 

Related to OIE release.

### Resolves:

* [OKTA-386724](https://oktainc.atlassian.net/browse/OKTA-386724)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -754,6 +754,27 @@ Specifies a set of Groups whose Users are to be included or excluded
     }
   }
 ```
+#### UserType Condition object
+
+Specifies a set of UserTypes are to be included or excluded. You can use the [UserTypes API](/docs/reference/api/user-types/) to manage UserTypes.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:759]: "Specifies which User Types to include and/or exclude"

And "User Types" should link to further information about User Types.

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 2]
```diff
@@ -754,6 +754,27 @@ Specifies a set of Groups whose Users are to be included or excluded
...
+
+| Parameter | Description                  | Data Type | Required |
+| ---       | ---                          | ---       | ---      |
+| include   | The UserTypes to be included | Array     | Yes      |
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:763]:
```suggestion
| include   | The User Types to be included | Array     | Yes      |
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 3]
```diff
@@ -754,6 +754,27 @@ Specifies a set of Groups whose Users are to be included or excluded
...
+| exclude   | The UserTypes to be excluded | Array     | Yes      |
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:764]:
```suggestion
| exclude   | The User Types to be excluded | Array     | Yes      |
```

**Review by jakubvul-okta — DISMISSED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 4]
```diff
@@ -754,6 +754,31 @@ Specifies a set of Groups whose Users are to be included or excluded
     }
   }
 ```
+#### UserType Condition object
+
+<ApiLifecycle access="ie" /><br>
+
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:761]:
```suggestion
> **Note:** This feature is only available as a part of the Okta Identity Engine. Contact [Support](mailto:dev-inquiries@okta.com) for more information.
```

**Review by susanharper-okta — COMMENTED** [review]
just one thing

**Review by susanharper-okta — APPROVED** [review]
LGTM! 🚢

**Pull request merged**